dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/AlertDao.java

```diff
@@ -81,9 +89,16 @@ public class AlertDao {
      * @return add alert result
      */
     public int addAlert(Alert alert) {
+        if (null == alert.getAlertGroupId() || NumberUtils.INTEGER_ZERO.equals(alert.getAlertGroupId())) {
```

Comment:
Why use `NumberUtils.INTEGER_ZERO` rather than `0 ==alert.getAlertGroupId()` here?

Reply:
because I want to reduce to use magic value